### PR TITLE
Made container logs tail value come from .env file

### PR DESCRIPTION
**Pull request by elraphty**
This feature allows the value of the container logs tail will come from a .env file
@Evanfeenstra @kevkevinpal

### CODE REVIEW

File: src/dock.rs [hunk 1]
```diff
@@ -69,10 +70,13 @@ pub async fn remove_container(docker: &Docker, id: &str) -> Result<()> {
 }
 
 pub async fn container_logs(docker: &Docker, name: &str) -> Vec<String> {
+    let tail_name = "LOGS_TAIL_VALUE";
```
Comment (kevkevinpal) [inline src/dock.rs:73]: maybe renaming to `LOG_TAIL_LENGTH` might make more sense

Comment (elraphty) [reply src/dock.rs:73]: Ok @kevkevinpal

Comment (elraphty) [reply src/dock.rs]: @kevkevinpal I have made the change

**Comment by Evanfeenstra** [comment]
@elraphty looks good. Good to have a default so the env var is optional

**Pull request merged**